src/main/java/com/google/sps/data/PlaceReviews.java

```diff
@@ -99,7 +99,7 @@ public void addRating(double rating) {
    */
   public boolean reviewedBy(String person) {
     // TODO: Use datastore querying to search for this
-    return true;
+    return false;
```

Comment:
I would rather you raise an exception here "NonImplementedError" and try/catch it wherever you use this function until you solve the TODO. It can be confusing if someone else sees this function being called and is not obvious it's hard hard-coded.